### PR TITLE
Clean old discovery metadata aerospike-namespaces

### PR DESCRIPTION
**Pull request by damsallem**

### CODE REVIEW

File: configs/aerospike_config.yaml [hunk 1]
```diff
@@ -26,10 +26,6 @@ client_config:
   # Skip ssl verification
   tls_skip_verify: false 
   ### Probe discovery configuration ###
-  # The key to discover Aerospike's namespaces through service discovery
-  # Set it to "" to instead discover namespaces automatically via the info command
-  # DEPRECATED since 02/19/2024
-  namespace_meta_key: "aerospike-namespaces"
   # The key prefix to discover Aerospike's namespaces through service discovery
   # old "namespace_meta_key" has been DEPRECATED because of 512 bytes limitation of the value in consul.
```
Comment (pgoron) [inline configs/aerospike_config.yaml:30]: I don't think keeping this comment bring useful information (it is in commit messages)

File: pkg/aerospike/discovery.go [hunk 2]
```diff
@@ -78,45 +78,26 @@ func (conf *AerospikeProbeConfig) generateNamespacedEndpointsFromEntry(logger lo
...
-			fallback = true
-			break
+			continue
 		}
-		// if ready is at false, then iterate to the next MetaData and try to resolve other namespaces
 		if !ready {
 			continue
 		}
-		ns := metaKey[len(conf.AerospikeEndpointConfig.NamespaceMetaKeyPrefix):] // MetaKey is like : "aerospike-monitoring-closeststore"
+		// MetaKey is like : "aerospike-monitoring-closeststore"
```
Comment (pgoron) [inline pkg/aerospike/discovery.go:94]: I would remove reference to a real namespace, use foobar instead

**Review by pgoron — CHANGES_REQUESTED** [review]

File: configs/aerospike_config.yaml (hunk 1 above)
Comment (damsallem) [reply configs/aerospike_config.yaml:30]: Absolutely, patching it !

File: pkg/aerospike/discovery.go (hunk 2 above)
Comment (damsallem) [reply pkg/aerospike/discovery.go]: Done !

**Review by pgoron — APPROVED** [review]

**Pull request merged**